src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -202,7 +208,7 @@ const styles = StyleSheet.create({
   },
   container: {
     backgroundColor: COLORS.default_signed_secondary,
-    borderRadius: 15,
+    borderRadius: 16,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The slight increase in the `borderRadius` of the container from 15 to 16 is a minor change and doesn't seem to have any significant impact on the overall design or functionality.
```diff
- borderRadius: 15,
+ borderRadius: 16,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->